src/main/java/net/backupcup/mcde/mixin/ItemStackMixin.java

```diff
@@ -124,7 +124,7 @@ public abstract class ItemStackMixin {
     }
 
     @Inject(method = "hasGlint", at = @At("HEAD"), cancellable = true)
-    private void mcde$hasEnchantments(CallbackInfoReturnable<Boolean> cir) {
+    private void mcde$hasGlint(CallbackInfoReturnable<Boolean> cir) {
```

Comment:
The name change from `hasEnchantments` to `hasGlint`. Why have you chosen this nomenclature? Glint is often used as a Mojmap term whereas Enchantments is used as a Yarn term. Unless I am misunderstanding something with this change, it is best to keep to the mapping standard nomenclature unless there is a good reason to depart from it.

Reply:
This is just for consistency, this is injected into `hasGlint` so it can become confusing if this is called like already existing `hasEnchantments` method while actually injecting to another one.